src/functions.pl

```diff
@@ -644,7 +644,10 @@ sub initMapChangeVars {
 	undef %items;
 	undef %spells;
 	undef %incomingParty;
+	
 	undef %talk;
+	AI::clear("NPC");
```

Comment:
Because the task will only be cleaned in the first AI cycle after the mapchange, this can cause major bugs if another npc conversation starts before that, like a npc that starts conversation as soon as the character enters a certain map. (eg. Payon palace guards)